modules/serverless-kn-cli-plugins.adoc

```diff
@@ -0,0 +1,14 @@
+// Module is included in the following assemblies:
+//
+// serverless/installing-knative-client.adoc
+
+[id="serverless-kn-plugins_{context}"]
+= `kn` plugins
+
+The Red Hat distribution of the `kn` command contains extensions of core `kn`, called plugins. Their main use is extending `kn` to use additional commands provided on {ServerlessProductName}, for example, the Kafka stack API. Main `kn` functionality and `kn` plugins are used in the same way.
+
+Currently, Red Hat provides the `kn kafka` and the `kn func` plugins.
```

Comment:
I'd call out that kafka is tech preview and kn func is dev preview, and maybe a link to the Functions dev preview docs https://openshift-knative.github.io/docs/docs/functions/about-functions.html

Reply:
Done, could you PTAL at the format? Mentioned it "in passing" in this paragraph, but inserted a proper TP note in the kn kafka plugin section proper.

Reply:
Ignore me, lets leave functions out of this altogether for now until it's tech preview please. See later note.

Reply:
Removed all mentions of functions.